db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
```

Comment:
Czy rozumie Pan przez to usunięcie dodania pola "competence" i zmianę numeru wersji na początku? Czy coś jeszcze?